Apps/W1/HybridGP/app/src/pages/HybridCloudWizardExtension.PageExt.al

```diff
@@ -73,28 +73,9 @@ pageextension 4014 "Hybrid Cloud Wizard Extension" extends "Hybrid Cloud Setup W
         {
             trigger OnAfterAction()
             begin
-                if Rec."Product ID" = 'DynamicsGP' then begin
-                    CleanupSettingsForCompaniesPreviouslyMigrated();
+                if Rec."Product ID" = 'DynamicsGP' then
                     Page.Run(Page::"GP Migration Configuration");
-                end;
             end;
-
         }
     }
-
-    local procedure CleanupSettingsForCompaniesPreviouslyMigrated()
```

Comment:
We need to test this well. It should include enabling and disabling the cloud migration. We should also test connecting a different DB to bring in a company